linux/meshd-nl80211.c

```diff
@@ -1303,6 +1306,9 @@ meshd_parse_libconfig (struct config_setting_t *meshd_section,
     CONFIG_LOOKUP(mcast-rate, mcast_rate);
     CONFIG_LOOKUP(beacon-interval,beacon_interval);
 
+    config_setting_lookup_int(meshd_section, "protected-management-frames",
+            (config_int_t *) &config->protected_management_frames);
```

Comment:
Should we make it something more simple in the authsae configuration, like pmf, similar to wpa_supp?